### PR TITLE
Create UsdPreviewSurface roughness test asset

### PR DESCRIPTION
**Pull request by jcowles**
See readme.md for details

### CODE REVIEW

File: test_assets/RoughnessTest/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+![screenshot](screenshots/usdview-20.08.png)  
+_usdview 20-08_
+
+## Description
+
+This file contains six meshes, the first three specify roughness values of 0.0, 0.33, and 0.66 specified via a texture.
+The next three meshes similarly specify roughness, however the value is authored as a constant on the UsdPreviewSurface material.
+
+Specification: https://graphics.pixar.com/usd/release/spec_usdpreviewsurface.html#transform2d
```
Comment (meshula) [inline test_assets/RoughnessTest/readme.md:13]:
```suggestion
Specification: https://graphics.pixar.com/usd/release/spec_usdpreviewsurface.html#roughness-vs-glossiness

**Review by meshula — COMMENTED** [review]
Having three different colors makes it possibly tricky to perceive the differences between the roughness levels ~ would it make sense to make them all a neutral gray? Or to make the numbers gray and distribute the colors across the horizontal axis so to speak?

File: test_assets/RoughnessTest/readme.md (hunk 1 above)
Comment (jcowles) [reply test_assets/RoughnessTest/readme.md]: Good catch!

**Comment by jcowles** [comment]
FYI, I'm also looking into making this asset a little lighter weight, it has quite a bit of geometry

**Comment by jcowles** [comment]
> Having three different colors makes it possibly tricky to perceive the differences between the roughness levels ~ would it make sense to make them all a neutral gray? Or to make the numbers gray and distribute the colors across the horizontal axis so to speak?

Will give it a shot

**Comment by hybridherbst** [comment]
Is there a particular reason for not including roughness=1.0?

**Comment by jcowles** [comment]
@hybridherbst it's more of a test to just see that values match, I don't think roughness 1.0 would add much to the three other values - do you have a specific issue in mind?

**Review by meshula — APPROVED** [review]
lgtm

**Comment by hybridherbst** [comment]
It's always good to see behavior at extremes of values. E.g. "what's the maximum or minimum roughness and metallic I can display in that viewer" is a question that comes up often. I think that's also why often these are combined into a matrix of sorts. The related glTF test asset is https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/MetalRoughSpheres.

One example of differences one can come across is under LDR or HDR lighting, see https://modelviewer.dev/fidelity/#khronos-MetalRoughSpheres-HDR and the LDR below it.

At least from my perspective I'm always thinking about "what can I learn from that asset / in which situations would I choose that particular test asset to test something".

**Comment by jcowles** [comment]
@hybridherbst oh I see what you mean, but this test isn't a BRDF test, this test is specifically about getting the right value if it comes from texture or constant value.

The test you pointed out is a great one for testing the actual BRDF, which we should also have

**Pull request merged**